From 05433a8b4bc65af592726c3ed482b9770526a6e1 Mon Sep 17 00:00:00 2001
From: "Luis Gustavo S. Barreto" <gustavo@ossystems.com.br>
Date: Wed, 6 Apr 2016 18:34:58 +0000
Subject: [PATCH 1/3] Fix package name

The properties of Exiv2 package are not being set properly
due to incorrect package name.

Upstream-Status: Pending

Signed-off-by: Luis Gustavo S. Barreto <gustavo@ossystems.com.br>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0f528ae..a18ca16 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -69,7 +69,7 @@ find_package(PNG)
 set_package_properties(PNG PROPERTIES URL "http://www.libpng.org" DESCRIPTION "PNG image manipulation support" TYPE REQUIRED)
 
 find_package(Exiv2)
-set_package_properties(EXIV2 PROPERTIES URL "http://www.exiv2.org" DESCRIPTION "image metadata support" TYPE REQUIRED)
+set_package_properties(Exiv2 PROPERTIES URL "http://www.exiv2.org" DESCRIPTION "image metadata support" TYPE REQUIRED)
 
 find_package(KF5Kipi)
 if (KF5Kipi_FOUND)
-- 
2.1.4

